### PR TITLE
Missing ManagementPointNetworkType param

### PR DESCRIPTION
**Pull request by DMillerF**
Adding in documentation around the ManagementPointNetworkType parameter for this command.

### CODE REVIEW

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md [hunk 1]
```diff
@@ -168,6 +168,29 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ManagementPointNetworkType
+Specifies the network configuration used to determine IP address settings.
+
+The acceptable values for this parameter are:
+
+```yaml
```
Comment (chasewilson) [inline docset/winserver2019-ps/failoverclusters/New-Cluster.md:176]: I would suggest against using a yaml block to represent the options. I would follow something more like the `AdministrativeAccessPoint` template and use  a bulleted list.

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md [hunk 2]
```diff
@@ -168,6 +168,29 @@ Accept pipeline input: False
...
+Singleton: Uses the traditional method of DHCP or static IP address.
```
Comment (chasewilson) [inline docset/winserver2019-ps/failoverclusters/New-Cluster.md:177]:
```suggestion
Singleton: The traditional method of DHCP or static IP address. (Default setting)
```

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md [hunk 3]
```diff
@@ -168,6 +168,29 @@ Accept pipeline input: False
...
+Distributed: Use a Distributed Network Name by using Node IP addresses.
+Automatic: Uses detection to determine the appropriate setting. If SQL Server is running in Azure, uses Distributed. If SQL Server is running on-premises, uses Singleton (default setting).
```
Comment (chasewilson) [inline docset/winserver2019-ps/failoverclusters/New-Cluster.md:179]:
```suggestion
Automatic: Automatically detects the appropriate setting. If SQL Server is running in Azure, uses `Distributed`. If SQL Server is running on-premises, uses `Singleton`.
```

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md [hunk 4]
```diff
@@ -168,6 +168,29 @@ Accept pipeline input: False
...
+```
+
+```yaml
+Type: AdminAccessPointResType
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
```
Comment (chasewilson) [inline docset/winserver2019-ps/failoverclusters/New-Cluster.md:189]:
```suggestion
Default value: Singleton
```

Double check this information to ensure accuracy. I'm making this changes based on your changes in the list of values above.

**Review by chasewilson — CHANGES_REQUESTED** [review]
Just a few things. A couple for easier reading. one for correct information* and one for formatting change. 

* A long as the information is correct.

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md (hunk 1 above)
Comment (PatAltimore) [reply docset/winserver2019-ps/failoverclusters/New-Cluster.md:176]: I also agree with removing the yaml block.

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md (hunk 4 above)
Comment (PatAltimore) [reply docset/winserver2019-ps/failoverclusters/New-Cluster.md:189]: When I used PlatyPS for WS2022, the default value was None.

Comment (DMillerF) [reply docset/winserver2019-ps/failoverclusters/New-Cluster.md:189]: I believe that Automatic is the default value for this parameter, according to the few sources I've seen.

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md [hunk 5]
```diff
@@ -168,6 +168,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ManagementPointNetworkType
+Specifies the network configuration used to determine IP address settings.
+
+The acceptable values for this parameter are:
+
+- Automatic: Automatically detects the appropriate setting. If SQL Server is running in Azure, uses `Distributed`. If SQL Server is running on-premises, uses `Singleton`. (Default Setting)
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/failoverclusters/New-Cluster.md:176]:
```suggestion
- Automatic: Automatically detects the appropriate setting. If SQL Server is running in Azure, uses `Distributed`. If SQL Server is running on-premises, uses `Singleton`. (Default setting.)
```

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md [hunk 6]
```diff
@@ -168,6 +168,27 @@ Accept pipeline input: False
...
+- Singleton: The traditional method of DHCP or static IP address.
+- Distributed: Use a Distributed Network Name by using Node IP addresses.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/failoverclusters/New-Cluster.md:178]:
```suggestion
- Distributed: Uses a Distributed Network Name by using Node IP addresses.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor suggestions.

**Comment by DMillerF** [comment]
@chasewilson Changes should be added, let me know if you see anything else.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md [hunk 7]
```diff
@@ -18,7 +18,7 @@ Creates a new failover cluster.
 ```
 New-Cluster [-Name] <String> [-Node <StringCollection>] [-StaticAddress <StringCollection>]
  [-IgnoreNetwork <StringCollection>] [-NoStorage] [-S2D] [-AdministrativeAccessPoint <AdminAccessPoint>]
- [-Force] [<CommonParameters>]
+ [-Force] [-ManagementPointNetworkType {Automatic | Singleton | Distributed}] [<CommonParameters>]
```
Comment (get-itips) [inline docset/winserver2019-ps/failoverclusters/New-Cluster.md:21]:
```suggestion
 [-Force] [-ManagementPointNetworkType <AdminAccessPointResType>] [<CommonParameters>]
```

**Review by dariomws — APPROVED** [review]
I'm ok after the suggestions. Thanks!

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md (hunk 7 above)
Comment (chasewilson) [reply docset/winserver2019-ps/failoverclusters/New-Cluster.md:21]: At least as far as I can find examining the cmdlet in PowerShell, the defined parameter type is `object` so I don't think this change would be correct.

```powershell
(Get-Command New-Cluster).Parameters.ManagementPointNetworkType.ParameterType

IsPublic IsSerial Name                                     BaseType                                                     
-------- -------- ----                                     --------
True     True     Object                                                                                                                                                                             
```

**Review by chasewilson — APPROVED** [review]
Aside from the single casing fix from @JohanFreelancer9, I think this looks good.

File: docset/winserver2019-ps/failoverclusters/New-Cluster.md (hunk 7 above)
Comment (get-itips) [reply docset/winserver2019-ps/failoverclusters/New-Cluster.md:21]: The parameter type is indeed AdminAccessPointResType according to PlatyPs:

image

Comment (get-itips) [reply docset/winserver2019-ps/failoverclusters/New-Cluster.md:21]: Let me know what you think @chasewilson  but we usually honor PlatyPs

Comment (chasewilson) [reply docset/winserver2019-ps/failoverclusters/New-Cluster.md:21]: Yup usually the best thing to do. Good by me

**Review by tseward — APPROVED** [review]

**Comment by sdwheeler** [comment]
@PatAltimore Is this change included in your PR?

**Comment by PatAltimore** [comment]
> @PatAltimore Is this change included in your PR?

Thanks @sdwheeler. I updated the 2022 PR to match this change.

**Pull request merged**